rflx/model/message.py

```diff
@@ -1310,7 +1333,7 @@ def _verify_use_of_literals(self) -> None:
                     ]
                 )
 
-    def _verify_expression_types(self) -> None:
+    def _verify_expression_types(self) -> None:  # pylint: disable=too-many-branches
```

Comment:
We should rather refactor the function instead of just disabling the check. The "too-many-branches" warning indicates the function is too complex.

Reply:
Fixed.